packages/build/config/tsconfig.common.json

```diff
@@ -5,7 +5,6 @@
     "experimentalDecorators": true,
     "noImplicitAny": true,
     "strictNullChecks": true,
-
```

Comment:
JSON files should not be linted by prettier. I find this empty line as very useful to delimit language-related flags from project-related settings.

Could you please revert this change?